### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by julianoes**
Follow up to #1887.

### CODE REVIEW

File: scripts/format_xml.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ OPTIND=1         # Reset in case getopts has been used previously in the shell.
 
 # Initialize variables
 mode="format"
-xml_dir="."
+xml_dir="message_definitions"
```
Comment (hamishwillee) [inline scripts/format_xml.sh:9]: So was this not working at all before?

Comment (julianoes) [reply scripts/format_xml.sh:9]: It just tried to fix files inside pymavlink: https://github.com/mavlink/mavlink/runs/8219253471?check_suite_focus=true#step:4:36

**Comment by julianoes** [comment]
Ok, this can be merged, I'd say.

**Comment by hamishwillee** [comment]
OK, on you :-)

**Pull request merged**